components/auth/onboarding-form.tsx

```diff
@@ -44,13 +44,14 @@ export function SignUpForm({
     }
 
     try {
-      const { publicKey, wrappedPrivateKey, salt } =
+      const { publicKey, wrappedPrivateKey, salt, generateAndWrapVaultKey } =
```

Comment:
This variable needs to be renamed too...

```suggestion
      const { publicKey, wrappedPrivateKey, salt, wrappedDefaultVaultKey } =
```